### ISSUE TITLE
Windows 10 build 11082: NVDA does not work properly with UIA controls which results in COM Error thrown when exiting NVDA

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#5617]
Hi,

This was observed with Windows 10 build 11082 (released December 16, 2015):

NVDA does not work properly with UIA objects in Windows 10. Some of the problems observed are:

* NVDA announces "Cortana window" when attempting to open Start menu.
* In File Explorer, Settings and other apps where lists are present, NVDA does not announce item count and/or announces list entries twice as if UIA isn't loaded.
* When toasts appear, NvDA does not announce them, and looking at developer info shows that it is an IAccessible object when we know for certain that it is a UIA control.

This often results in the following traceback when you exit NVDA:

ERROR - stderr (06:53:14):
Exception in thread Thread-4:
Traceback (most recent call last):
  File "threading.pyc", line 810, in __bootstrap_inner
  File "threading.pyc", line 763, in run
  File "_UIAHandler.pyc", line 180, in MTAThreadFunc
COMError: (-2147024882, 'Not enough storage is available to complete this operation.', (None, None, None, 0, None))

Setup and additional information:
* NVDA version: next.12987, latest master snapshots, 2015.4.
* OS: Windows 10.0.11082 fast ring build
* Consistently reproducible: Yes.
* Confirmed by other Insiders: Yes.
* Feedback sent to Microsoft: Not yet, still gathering information before filing a feedback.
* Known workaround: Logging off or restarting a computer sometimes fixes this issue.
* Occurs with Narrator and other screen readers: At least this wasn't observed with Narrator.

Thanks.

**Comment by mwcampbell** [comment]
The same problem is happening with System Access from Serotek, but not with Narrator. I've written to Microsoft about it.

**Comment by beqabeqa473** [comment]
is some hints to fix this issue?

**Comment by jcsteh** [comment]
I'm told logging out and back in might help, but as for a proper fix, 
you'll have to wait until Microsoft fixes it; it's most definitely a 
Windows bug. You could always revert to build 10586.

**Comment by beqabeqa473** [comment]
unfortunately, reverting process is not accessible, very strange, even object navigation doesn't work.

**Comment by beqabeqa473** [comment]
modern apps aren't accessible too, log says about not enough memory to perform operation

**Comment by dreinn** [comment]
I have this problem also with a pc of a friend of mine.
I can add that if you open notepad, NVDA reads "blank" when you try to read menu.

Windows10 home 64 bit, unfortunately I don't know the build...

**Comment by jcsteh** [comment]
This is fixed in Windows 10 build 14251.
